<launch>

    <arg name="publish_robot_source_odom_tf" default="false"/>


    <!--move_base-->
    <include file="$(find basecontrol)/launch/movebase.launch">
        <arg name="map_filename" value="/home/xcy/aworkspace/catkin_simulator/src/basecontrol/map/gazebomap.pgm"/>
    </include>
    
    <!-- map related-->
    <!--node name="rviz" pkg="rviz" type="rviz" args="-d $(find basecontrol)/config/rviz/gazebo.rviz"/-->

    <node name="simpleNav" pkg="basecontrol" type="simpleNav" output="screen">
        <param name="base_foot_print" value="base_link"/>
        <param name="odom_frame" value="odom"/>
        <param name="publish_tf" value="$(arg publish_robot_source_odom_tf)"/>
        <param name="serial_addr" value="/dev/ttyUSB0"/>
    </node>

</launch>
